fastlane/.gitignore

```diff
@@ -3,5 +3,6 @@
 /screenshots/*/*.png
 /screenshots/screenshots.html
 /screenshots/report.xml
+/metadata/dw/
```

Comment:
This is already covered by the ´/metadata/` line below

Reply:
yeah. that line is removed in a different PR.